packages/web/src/relay/createQueryRendererModern.tsx

```diff
@@ -14,9 +13,13 @@ type Config = {
   hideSplash?: boolean,
 };
 
-export default function createQueryRenderer(
-  FragmentComponent,
-  Component,
+type FragmentComponentTypes = {
+  query?: Variables;
+}
+
+export default function createQueryRenderer<P>(
+  FragmentComponent: RelayContainer<P> & FragmentComponentTypes,
```

Comment:
Not sure what is the correct type here.
Ts still arguing with me about it.

Reply:
what is the error?

Reply:
```
Type '{ query: Variables; children?: ReactNode; }' is not assignable to type 'IntrinsicAttributes & MappedFragmentProps<Pick<P, Exclude<keyof P, "relay">>> & ComponentRef & { children?: ReactNode; }'.
  Property 'query' does not exist on type 'IntrinsicAttributes & MappedFragmentProps<Pick<P, Exclude<keyof P, "relay">>> & ComponentRef & { children?: ReactNode; }'.ts(2322)
```

Reply:
each fragment will have different props

Reply:
so any in this scenario is reasonable?

Reply:
Any updates on passing the test? 
Talking about `Type '{ query: Variables; children?: ReactNode; }' is not assignable to type 'IntrinsicAttributes & MappedFragmentProps<Pick<P, Exclude<keyof P, "relay">>> & ComponentRef & { children?: ReactNode; }'.
  Property 'query' does not exist on type 'IntrinsicAttributes & MappedFragmentProps<Pick<P, Exclude<keyof P, "relay">>> & ComponentRef & { children?: ReactNode; }'.ts(2322)`

Reply:
I'm trying to fix some others on #82, but i'm having problems with jest and relay. If you want to help, go give it a try :)

Reply:
Ill submit a PR soon hopefully.